orders/models.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
```

Comment:
Rather than importing `User`, let's import `get_user_model` and use that function instead. This way, if we add a custom model for users, the system can handle it.